guides/source/testing.md

```diff
@@ -1398,6 +1398,56 @@ class ProfileControllerTest < ActionDispatch::IntegrationTest
 end
 ```
 
+#### Extracting Files
+
+If you find your helpers are clutering `test_helper.rb`, you can extract them into separate files. One good place to store them is `lib/test`.
+
+```ruby
+# lib/test/multiple_assertions.rb
+module MultipleAssertions
+  def assert_multiple_of_fourty_two(number)
+    assert (number % 42 == 0), 'expected #{number} to be a multiple of 42'
+  end
+end
+```
+
+These helpers can then be explicitly required as needed and included as needed
+
+```ruby
+require 'test_helper'
+require Rails.root.join('lib', 'test', 'multiple_assertions')
```

Comment:
Should we be promoting using pathnames with `require` or appending the `lib/test` to the `$LOAD_PATH`?

Reply:
`lib` is actually already in the load_path so we can just use regular strings instead of pathnames here.

Reply:
So I'd replace this with
```ruby
require 'test/multiple_assertions`
```

If so, perhaps the directory should be renamed to something like `test_support`, `test_helpers` or something? 🤔

Reply:
Correct. The rails root isn't part of the load paths, so I don't think its entirely misleading.

Reply:
Done 👍